core/services/ardupilot_manager/mavlink_proxy/Endpoint.py

```diff
@@ -47,6 +49,22 @@ def from_str(self, pattern: str) -> "Endpoint":
 
         return self
 
+    def from_dict(self, pattern: Dict[str, str]) -> "Endpoint":
+        if not ("connType" in pattern and "place" in pattern):
+            raise RuntimeError("Wrong format for endpoint creation.")
```

Comment:
`Wrong format for endpoint creation, missing the following: {{"connType", "place"}.difference(pattern.keys())}`